proxy/fadein.go

```diff
@@ -5,19 +5,19 @@ import (
 	"math/rand"
 	"time"
 
-	"github.com/zalando/skipper/eskip"
 	"github.com/zalando/skipper/routing"
 )
 
-func returnFast(rt *routing.Route) bool {
-	if rt.BackendType != eskip.LBBackend {
```

Comment:
Why did you remove this condition?

Reply:
The filterFadeIn function is called only by `selectEndpoint` from proxy.go. However, the `selectEndpoint` is called only for `eskip.LBBackend`, see https://github.com/zalando/skipper/blob/v0.19.32/proxy/proxy.go#L505 .

This fact means there is no sense to check BackendType one more time, is was already checked before.